### DISCUSSION TITLE
How to use "subfolders" in public-urls?

### DISCUSSION THREAD

**Discussion by pawelklimczyk** [satnaing/astro-paper#192] [Q&A]
I would like to organize my blog posts into "year" or "year/month" subfolders.

the final result would be one of following:
-  postSlug defined as (post-123) in file post123.md in folder /2023/12/post-123 and public-url as `https://<blog-url>/articles/2023/12/post-123`
-  postSlug defined as (post-123) in file post123.md in folder /2023 and public-url as `https://<blog-url>/articles/2023/post-123`

So basically public-url matches postSlug and generates url sub-path from directory names structure

After several tries and errors i have to sek for help 😛
